app/src/test/java/de/westnordost/streetcomplete/quests/opening_hours/AddOpeningHoursTest.kt

```diff
@@ -64,6 +99,15 @@ class AddOpeningHoursTest {
         )
     }
 
+    @Test fun `apply no opening hours sign answer when there was an always open answer before`() {
+        questType.verifyAnswer(
+            mapOf("opening_hours" to "24/7"),
+            NoOpeningHoursSign,
+            StringMapEntryAdd("opening_hours:signed", "no"),
+            StringMapEntryAdd("check_date:opening_hours", LocalDate.now().toCheckDateString())
+        )
+    }
+
```

Comment:
I've just realised that all apart from this one are ineligible for this quest as they'll be filtered out for being unsigned already.

I should be able to apply most to https://github.com/streetcomplete/StreetComplete/blob/master/app/src/test/java/de/westnordost/streetcomplete/quests/opening_hours_signed/CheckOpeningHoursSignedTest.kt though.

Reply:
Delete them then? Or move them to CheckOpeningHoursSignedTest?

Reply:
I've migrated some and reworked others and deleted some others to make them all relevant and appropriate to the various quests.

As part of #4276 they could be extended further/rewritten.